API.md

```diff
@@ -2,9 +2,27 @@
 
 This page documents the Collection Instrument service API endpoints.
 
-### Encrypt and upload a collection exercise
+### Encrypt and upload a collection exercise with RU
 
-* `POST /collection-instrument-api/1.0.2/upload/cb0711c3-0ac8-41d3-ae0e-567e5ea1ef87/test.xls` (with a file in the data of the POST)
+* `POST /collection-instrument-api/1.0.2/upload/cb0711c3-0ac8-41d3-ae0e-567e5ea1ef87/999` (with a file in the data of the POST)
```

Comment:
I think it would be better to write this example url as a general case, `/collection-instrument-api/1.0.2/upload/<exercise_id>/<RU_id>`
It's not immediately obvious what the uuid is representing

Reply:
Yep, your right, does make it a bit less ambiguous, will update all of them